ui/components/YamlView.tsx

```diff
@@ -53,18 +60,19 @@ function UnstyledYamlView({ yaml, object, className, theme }: YamlViewProps) {
 
   return (
     <div className={className}>
-      <YamlHeader>
-        {headerText}
-        {headerText && (
+      {headerText && (
+        <YamlHeader>
+          {headerText}
           <CopyToClipboard
             value={headerText}
             className="yaml-copy"
             size="small"
           />
-        )}
-      </YamlHeader>
+        </YamlHeader>
+      )}
+
       <SyntaxHighlighter
-        language="yaml"
```

Comment:
These changes in YamlView don't seem to be used elsewhere? Are they necessary?

Reply:
This meant to be used in displaying the `violatingEntities` in Violation details and Policy code in PolicyDetails and currently we dropped `violatingEntities` and sowed as a modal .
beside this will give the ability to change it when we want .